src/BusinessNotificationCenter.jsx

```diff
@@ -0,0 +1,120 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import { SearchIcon, PlusSquareIcon, BellIcon, HamburgerIcon } from '@chakra-ui/icons';
+import {
+  Text,
+  InputGroup,
+  InputLeftElement,
+  Icon,
+  Menu,
+  MenuButton,
+  Button,
+  Input,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@chakra-ui/react';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  useDisclosure,
+} from '@chakra-ui/react';
+
+const App = () => {
```

Comment:
you guys are actually jits for calling the main component that is exported in `BusinessNotificationCenter.jsx` to be `App`, we already have a file called App.jsx which is rendered by index.jsx. 

Instead of calling this and exporting this as App, call it BusinessNotificationCenter and export this and call it  into App.jsx, not main.jsx.